src/NuGet.Services.AzureSearch/SearchService/SearchTextBuilder.cs

```diff
@@ -16,6 +17,11 @@ public partial class SearchTextBuilder : ISearchTextBuilder
     {
         public const string MatchAllDocumentsQuery = "*";
         private static readonly char[] PackageIdSeparators = new[] { '.', '-', '_' };
+        private static readonly char[] TokenizationSeparators = new[] { '.', '-', '_', ',', ';' };
```

Comment:
I started by tokenizing on just package id separators, but I was able to improve the feedback score from  0.7285 to 0.7315 by adding `,`. I also added `;` arbitrarily, removing it doesn't affect the feedback score.

Reply:
There are other symbols that `PackageIdCustomAnalyzer` splits on. Why don't we do those ones too? How were `,` and `;` picked in addition to the ones that can be in package IDs?

Reply:
I went through data to see if including these characters could improve results. I found that almost all `;` were at the end of the query, which isn't helped by this change. As such, I've kept only `,`.